webpack.config.js

```diff
@@ -4,18 +4,23 @@ const path = require("path");
 const process = require("process");
 const webpack = require("webpack");
 const lodash = require("lodash");
-const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const Webpackbar = require("webpackbar");
-const ZipPlugin = require("zip-webpack-plugin");
 const headersCsp = require("./public/_headersCsp.json");
 
 // load ".env"
 dotenv.config();
 
+let scrivitoOrigin = "";
+if (process.env.CONTEXT === "production") {
+  scrivitoOrigin = process.env.URL;
+} else if (process.env.DEPLOY_PRIME_URL) {
+  scrivitoOrigin = process.env.DEPLOY_PRIME_URL;
+}
+
```

Comment:
FYI: Given that `scrivitoOrigin` is now only used in `webpackConfig` we can move this block back to its original place.